com.unity.render-pipelines.high-definition/Documentation~/HDRP-Camera.md

```diff
@@ -16,9 +16,10 @@ The HDRP Camera shares many properties with the [Standard Unity Camera](https://
 | **Background Color**            | Use the HDR color picker to select the color that the Camera uses to clear the screen before it renders a frame. The Camera uses this color if:You select **Color** from the **Background Type** drop-down.You select **Sky** from the **Background Type** drop-down and there is no valid sky for the Camera to use. |
 | **Culling Mask**                | Use the drop-down to set the Layer Mask that the Camera uses to exclude GameObjects from the rendering process. The Camera only renders Layers that you include in the Layer Mask. |
 | **Volume Layer Mask**           | Use the drop-down to set the Layer Mask that defines which Volumes affect this Camera. |
-| **Volume Anchor Override**      | Assign a Transform that the [Volume](Volumes.html) system uses to handle the position of this Camera. For example, if your application uses a third person view of a character, set this property to the character's Transform. The Camera then uses the post-processing and Scene settings for Volumes that the character enters.If you do not assign a Transform, the Camera uses its own Transform instead. |
+| **Volume Anchor Override**      | Assign a Transform that the [Volume](Volumes.html) system uses to handle the position of this Camera. For example, if your application uses a third person view of a character, set this property to the character's Transform. The Camera then uses the post-processing and Scene settings for Volumes that the character enters. If you do not assign a Transform, the Camera uses its own Transform instead. |
 | **Probe Layer Mask**            | Use the drop-down to set the Layer Mask that the Camera uses to exclude environment lights (light from Planar Reflection Probes and Reflection Probes). The Camera only uses Reflection Probes on Layers that you include in this Layer Mask. |
 | **Occlusion Culling**           | Enable the checkbox to make this Camera not render GameObjects that are not currently visible. For more information, see the [Occlusion Culling documentation](<https://docs.unity3d.com/Manual/OcclusionCulling.html>). |
+| **Exposure Target** | Game Object around which the [Auto Exposure](Override-Exposure.html) procedural mask is centered. |
```

Comment:
The GameObject to center the [Auto Exposure](Override-Exposure.md) procedural mask around.